tests/Fixtures/Integration/priority/class_keyword,fully_qualified_strict_types.test

```diff
@@ -0,0 +1,14 @@
+--TEST--
+Integration of fixers: class_keyword,fully_qualified_strict_types.
+--RULESET--
+{"class_keyword": true, "fully_qualified_strict_types": true}
```

Comment:
```suggestion
{"class_keyword": true, "fully_qualified_strict_types": {"import_symbols": true}}
```

Reply:
I tried that too, but it still fails with `Failed asserting that a NULL is not empty.`